spec/index.bs

```diff
@@ -796,6 +796,24 @@ interface mixin GPUObjectBase {
 };
 </script>
 
+<div algorithm>
+    <div data-timeline=content>
+        To <dfn abstract-op>create a new WebGPU object</dfn>({{GPUObjectBase}} |parent|,
+        interface |T|, {{GPUObjectDescriptorBase}} |descriptor|)
+        (where |T| extends {{GPUObjectBase}}):
+
+        1. Let |device| be |parent|.{{GPUObjectBase/[[device]]}}.
+        1. Let |object| be a new instance of |T|.
+        1. Let |internals| be a new (uninitialized) instance of the type of
+            |T|.`[[internals]]` (which may override {{GPUObjectBase}}.{{GPUObjectBase/[[internals]]}})
+            that is accessible only from the [=device timeline=] of |device|.
+        1. Set |object|.{{GPUObjectBase/[[device]]}} to |device|.
+        1. Set |object|.{{GPUObjectBase/[[internals]]}} to |internals|.
```

Comment:
In https://github.com/gpuweb/gpuweb/pull/3404/ you've updated the text to indicate that operations against `[[device]]` or `[[internals]]` on timelines other than the device timeline assert. I would expect that setting them initially still counts as an "operation". Does this need to happen on the device timeline?

Reply:
I reworded #3404 slightly.

Reply:
Still feels a little strange, but at least the "on the content of" distinction doesn't feel like it conflicts with this use case any more.